### ISSUE TITLE
Publish general accessibility acceptance criteria that all components should meet, and explain how to write component specific accessibility acceptance criteria

### ISSUE THREAD

**Issue by kellylee-gds** [alphagov/govuk-design-system#1781]
## What
Draft and publish accessibility acceptance criteria to be used when contributing new components to GOV.UK Design System.

## Why
Whilst building the Notification banner and Cookie banner components we used a template to check how components in the Design System should behave with various assistive technologies. We feel there is value publishing this information so it can be used by future contributors.

## Who needs to know about this
Hanna, Eoin

## Done when
- [x] Documentation scoped
- [x] Documentation drafted
- [x] Identified where documentation will live
- [x] We've asked a couple of FE devs or interaction designers to read through the doc and give us some feedback on it
- [x] 2i'd
- [ ] Published

**Comment by EoinShaughnessy** [comment]
This may change in future, but for now the likeliest location for this content is probably within https://github.com/alphagov/govuk-frontend/tree/main/docs/contributing.

**Comment by EoinShaughnessy** [comment]
Unless we wanted to add it here: https://design-system.service.gov.uk/community/?

**Comment by hannalaakso** [comment]
Early draft here: https://docs.google.com/document/d/133tfhMrCzpZizAOtggzgXGJJRCJKt7KR6M3WUWZekk0/edit

**Comment by EoinShaughnessy** [comment]
We've shared the doc with the rest of the Design System team for internal review.

**Comment by hannalaakso** [comment]
@vanitabarrett has kindly offered to do a frontend dev review of the draft.

**Comment by EoinShaughnessy** [comment]
We've updated the doc following feedback from the Frontend and accessibility communities. I've now sent the doc for 2i.

**Comment by EoinShaughnessy** [comment]
Doc has passed 2i. Will now publish it.
